addons/web/static/src/fields/field_tooltip.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="web.FieldTooltip" owl="1">
```

Comment:
is this template new in master-wowl? I'm a bit concerned that it could be used by legacy code

Reply:
it's new, the older one has another name